etc/kayobe/environments/stackhpc-sushy-baremetal/ansible/auto-setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: localhost
+  gather_facts: false
+
+  environment:
+    VIRTUAL_ENV: "{{ lookup('env', 'VIRTUAL_ENV') }}"
+    KAYOBE_CONFIG_PATH: "{{ lookup('env', 'KAYOBE_CONFIG_PATH') }}"
+
+  tasks:
+    - name: Run Sushy Emulator playbook
+      ansible.builtin.shell: |
+          source "$VIRTUAL_ENV/bin/activate"
+          kayobe playbook run "$KAYOBE_CONFIG_PATH/environments/stackhpc-sushy-baremetal/ansible/sushy-emulator.yml"
```

Comment:
Do we need this split into two, so we can discover the VM uuid, then configure sushi with the uuid we get given?